servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -0,0 +1,247 @@
+[[install_prerequisites]]
+= {ProductName} Installation Prerequisites
+
+Before you can install {ProductName}, you should already have met the following prerequisites:
+
+* Possess an active OpenShift Container Platform (OCP) subscription on your Red Hat account. Contact your sales representative for more information if you do not have a subscription.
+* Installed OpenShift Container Platform version 3.10 or higher.   For more information about the system and environment requirements, see the https://access.redhat.com/documentation/en-us/openshift_container_platform/3.10/html/installing_clusters/install-config-install-prerequisites#system-requirements[OpenShift Container Platform documentation].
+* Installed the version of the OpenShift Container Platform command line utility (the `oc` client tool) that matches your OCP version (for example if you have OCP 3.10 you must have the matching `oc` client version 3.10), and added it to your path.  For installation instructions, see the OpenShift Container Platform https://access.redhat.com/documentation/en-us/openshift_container_platform/3.10/html-single/cli_reference/#installing-the-cli[Command Line Reference] document.
+
+= Creating a Custom Resource File
+
+In order to deploy the {ProductShortName} control plane you need to deploy a custom resource.  A _custom resource_ is an object that extends the Kubernetes API or allows you to introduce your own API into a project or a cluster.  You define a custom resource as a yaml file that defines the object. Then you use the yaml file to create the object. The complete example below contains all of the supported parameters and will deploy {ProductName} {ProductVersion} images based on Red Hat Enterprise Linux (RHEL).  
+
+TIP:  Deploying an istio-installation file that includes all of the parameters will ensure that you have installed all of the Istio components required to complete the tutorials included in this document.
+
+.Full example istio-installation.yaml
+```
+apiVersion: "istio.openshift.com/v1alpha1"
+kind: "Installation"
+metadata:
+  name: "istio-installation"
+spec:
+  deployment_type: openshift
+  istio:
+    authentication: true
+    community: false
+    prefix: openshift-istio-tech-preview/
+    version: 0.1.0
+  jaeger:
+    prefix: distributed-tracing-tech-preview/
+    version: 1.6.0
+    elasticsearch_memory: 1Gi
+  launcher:
+    openshift:
+      user: user
+      password: password
+    github:
+      username: username
+      token: token
```

Comment:
The openshift and github values must be modified to reflect the user's openshift user/password chosen for running the launcher and also the user's github username/access token, it would be worth pointing this out.